docs/machine-learning/how-to-guides/load-data-model-builder.md

```diff
@@ -16,7 +16,7 @@ Learn how to load your training datasets from a file or a SQL Server database fo
 
 Model Builder limits the amount and type of data you can use for training models:
 
-- SQL Server data: 100,000 rows
+- SQL Server data: No size limit
 - CSV and TSV files: No size limit
 - Images: PNG and JPG only.
```

Comment:
Let's remove the limitations section since there aren't virtually any limitations. We can mention the fact that for files it only accepts TSV, CSV and semi-colon delimited files and for images only PNG and JPG are accepted.